pyscriptjs/tests/integration/test_py_runtime_config.py

```diff
@@ -0,0 +1,68 @@
+import os
+import tarfile
+import tempfile
+
+import pytest
+import requests
+
+from .support import PyScriptTest
+
+URL = "https://github.com/pyodide/pyodide/releases/download/0.20.0/pyodide-build-0.20.0.tar.bz2"
+TAR_NAME = "pyodide-build-0.20.0.tar.bz2"
+
+
+@pytest.fixture
+def tar_location(request):
+    val = request.config.cache.get("pyodide-0.20-tar", None)
+    if val is None:
+        response = requests.get(URL, stream=True)
+        TMP_DIR = tempfile.mkdtemp()
+        TMP_TAR_LOCATION = os.path.join(TMP_DIR, TAR_NAME)
+        with open(TMP_TAR_LOCATION, "wb") as f:
+            f.write(response.raw.read())
+        val = TMP_TAR_LOCATION
```

Comment:
veeeery nitpicking, but note that here you are loading 151MB in memory all at once, just to write them to disk. I'm sure there are better ways to save an http request to disk.
That said, it's not a blocker for this PR, but just wanted to say it as a general comment/suggestion for the future.

Reply:
Chunking perhaps?

Reply:
chunking is surely a good option.
I would have expected `requests` to have a direct way to save to disk but apparently there isn't.
A quick google reveals [this](https://reqbin.com/code/python/6mwlgbqa/python-requests-download-file-example).